core/modules/user/user.module

```diff
@@ -2775,13 +2775,20 @@ function user_role_revoke_permissions($role_name, array $permissions = array())
 }
 
 /**
- * List user administration filters that can be applied.
+ * Provided filters that could be applied to the user account listing.
+ *
+ * Same as with user_build_filter_query(), this function was used in the user
+ * account listing page in Drupal 7. Since that listing is provided by Views in
+ * Backdrop, this function here is slated for removal in Backdrop 2.0. Kept only
+ * for backwards-compatibility purposes in Backdrop 1.0.
+ *
+ * @todo Remove this function in Backdrop 2.0.
```

Comment:
Removed functions are first deprecated in the 1.x.

Reply:
@kiamlaluno should I add `@deprecated since 1.0` then? (plus a watchdog call)

Reply:
Those are changes for the other issue. The issue this PR should resolve is *user_filters() shows module and permission machine names rather than their human-readable labels, and tries to translate them*.

Reply:
Fair enough 👍🏼 ...I'll remove these then